### ISSUE TITLE
View: Searching network elements failing in some cases

### ISSUE THREAD

**Issue by jvwong** [PathwayCommons/app-ui#515]
## Environment info

- Browser version (client) : Version 65.0.3325.162 (Official Build) (64-bit), Firefox 58.0.2 (64-bit)
- Node.js version (server) : v8.8.1

## Current (buggy) behaviour

When I try to search for a node name I know exists within a pathway, the nodes are not highlighted at all.   

## Desired behaviour

I expect the node with a matching string to be highlighted.

## Minimum steps to reproduce

View ['Stabilization of p53 | Reactome'](http://localhost:3000/view?uri=http%3A%2F%2Fidentifiers.org%2Freactome%2FR-HSA-69541) and type 'TP53' in search then enter.

![image](https://user-images.githubusercontent.com/4706307/37468114-7256b730-2838-11e8-9377-ffa1c5b19f29.png)

Note the resulting console error:

```js
bundle.js:583 Uncaught TypeError: name.toUpperCase is not a function
    at bundle.js:583
    at evaluateNode (bundle.js:551)
    at bundle.js:582
    at Collection.filter (deps.js:81708)
    at searchNodes (bundle.js:581)
    at invokeFunc (deps.js:12339)
    at trailingEdge (deps.js:12388)
    at timerExpired (deps.js:12376)
``` 

Can also produce this error with ['Regulation of TP53 Activity through Methylation | Reactome'](http://localhost:3000/view?uri=http%3A%2F%2Fidentifiers.org%2Freactome%2FR-HSA-6804760)

**Comment by logmosier** [comment]
I am also getting this bug at [ATM pathway | pid](http://localhost:3000/view?uri=http%3A%2F%2Fpathwaycommons.org%2Fpc2%2FPathway_64ff47d79516caff57535f56c2018f5) when searching for ATM. 
Chrome: Version 64.0.3282.186 (Official Build) (64-bit)
Node js: v8.9.4

**Comment by jvwong** [comment]
Addressed in #517
